Sygic.Corona.Application/Commands/SendMorningAlertNotificationCommandHandler.cs

```diff
@@ -24,8 +24,10 @@ protected override async Task Handle(SendMorningAlertNotificationCommand request
                 if (daysLeft.HasValue)
                 {
                     string days = daysLeft.Value.Days.ToString();
-                    string messageText = 
-                        $"Váš zostávajúci počet dní v karanténe: {days}. Ďakujeme že dodržiavate domácu karanténu. Správajme sa zodpovedne voči blízkym a nášmu okoliu. MV SR";
+                    string messageText = !string.IsNullOrEmpty(request.Message)
+                        ? string.Format(request.Message, days) 
```

Comment:
You should handle the case of wrong message format (no {days} in the message)

Reply:
string.Format won't throw an exception if the message has no {days} argument, and it SHOULD throw an exception otherwise.

Reply:
string.Format won't throw an exception if the message has no {days} argument, and it SHOULD throw an exception if the string is not correctly formatted.